metro.config.js

```diff
@@ -13,5 +13,13 @@ module.exports = {
         inlineRequires: true,
       },
     }),
+    minifierConfig: {
```

Comment:
This exists so that we don't get profiled components showing up as `unknown` in spans.

Reply:
nice fix!